content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -109,25 +114,29 @@ $ export REMOTE_CLUSTER_NETWORK=network1
 
 ### Main cluster
 
-Create the main cluster's configuration. Replace the variables below with the cluster
-and network names chosen earlier. Pick **one** of the two options for cross-cluster
-control plane access and delete the configuration for the other two options.
+Create the main cluster's configuration. Pick one of the two options for cross-cluster
+control plane access.
+
+{{< tabset category-name="platform" >}}
+{{< tab name="istio-ingressgateway" category-value="istio-ingressgateway" >}}
 
 {{< text yaml >}}
 cat <<EOF> istio-main-cluster.yaml
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
   values:
-    # required if istiod is disabled
+    # selfSigned is required if Citadel is enabled (i.e. when values.global.istiod.enabled=false)
```

Comment:
If possible, avoid using "i.e." (https://developers.google.com/style/word-list#ie)

This note also applies to other instances of "i.e." in this document.